packages/cli/__tests__/daemon/__integration__/disable/__scripts__/daemon_disable_help.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+echo "================ daemon enable help ==============="
```

Comment:
Once again, this is help on disabling the daemon.

Reply:
yep.